avocado/utils/pci.py

```diff
@@ -129,6 +129,26 @@ def get_interfaces_in_pci_address(pci_address, pci_class):
                                                        interface))]
 
 
+def get_pci_class_name(pci_address):
+    """
+    Gets pci class name for given pci bus address
+
+    e.g: >>> pci.get_pci_class_name("0000:01:00.0")
+             'scsi_host'
+
+    :param pci_address: Any segment of a PCI address(1f, 0000:00:if, ...)
+
+    :return: class name for corresponding pci bus address
+    """
+    pci_class_dic = {'0104': 'scsi_host', '0c04': 'fc_host',
+                     '0200': 'net', '0108': 'nvme', '0280': 'net'}
+    pci_class_id = get_pci_prop(pci_address, "Class")
+    if pci_class_id not in pci_class_dic.iterkeys():
```

Comment:
Good, but unnecessary. The `if pci_class_id not in pci_class_dic` does the same via `__in__`. Would you please change it (as you need to adjust the `ValueError`, otherwise it's not a big deal, but it could be better optimized by future versions of python).